src/app/Schedule/SchedulePage.tsx

```diff
@@ -118,10 +135,11 @@ export default function SchedulePage({ currentShow }: SchedulePageProps) {
     show: ScheduleShow,
     dayName: string,
   ): boolean => {
-    if (!currentShow) return false;
```

Comment:
I think we can actually probably get rid of `currentShowRef`? I don't think it does anything...